frontend/components/modal/text/ModalTextEvent.vue

```diff
@@ -1,77 +1,25 @@
 <!-- SPDX-License-Identifier: AGPL-3.0-or-later -->
 <template>
   <ModalBase :modalName="modalName">
-    <Form
-      @submit="handleSubmit"
-      :schema="schema"
-      :initial-values="formData"
-      :submit-label="$t('i18n.components.modal._global.update_texts')"
-    >
-      <h2>
-        {{ $t("i18n.components.modal_text_event.edit_event_texts") }}
-      </h2>
-      <FormItem
-        v-slot="{ id, handleChange, handleBlur, errorMessage, value }"
-        :label="$t('i18n._global.description')"
-        name="description"
-        :required="true"
-      >
-        <FormTextArea
-          @input="handleChange"
-          @blur="handleBlur"
-          :id="id"
-          :value="value.value"
-          :hasError="!!errorMessage.value"
-        />
-      </FormItem>
-      <FormItem
-        v-slot="{ id, handleChange, handleBlur, errorMessage, value }"
-        :label="$t('i18n.components._global.participate')"
-        name="getInvolved"
-      >
-        <FormTextArea
-          @input="handleChange"
-          @blur="handleBlur"
-          :id="id"
-          :value="value.value"
-          :hasError="!!errorMessage.value"
-        />
-      </FormItem>
-      <FormItem
-        v-slot="{ id, handleChange, handleBlur, errorMessage, value }"
-        :label="$t('i18n.components.modal_text_event.offer_to_help_link')"
-        name="getInvolvedUrl"
-      >
-        <FormTextInput
-          @blur="handleBlur"
-          @update:modelValue="handleChange"
-          :id="id"
-          :modelValue="value.value as string"
-          :hasError="!!errorMessage.value"
-          :label="
-            $t('i18n.components.modal_text_event.offer_to_help_link_label')
-          "
-        />
-      </FormItem>
-    </Form>
+    <FormTextEntity
+      :formData="formData"
+      :handleSubmit="handleSubmit"
+      :submitLabel="'i18n.components.modal._global.update_texts'"
+      :title="'i18n.components.modal_text_event.edit_event_texts'"
+      :getInvolvedLabel="$t('i18n.components._global.participate')"
```

Comment:
Quick note on this, @OmkarSathish: We always like to have the `$t()` localization at the lowest level within components. @nicki182 and I are looking at this now and will move this into the form. Feel free to move the localization into the lowest component on the next issue 😊